components/inference-service/requirements.txt

```diff
@@ -1,4 +1,5 @@
 fastapi
 uvicorn
 aiofiles
-python-multipart
\ No newline at end of file
+python-multipart
+kubernetes==19.15.0
```

Comment:
Probably not significant in the initial phases, but we need a good plan to manage the underlying K8 requirements.  This is highlighted by pinning to a specific release.